bolero-generator/Cargo.toml

```diff
@@ -16,6 +16,7 @@ std = ["alloc", "either/use_std"]
 alloc = ["rand_core/alloc"]
 
 [dependencies]
+arbitrary = { version = "1.0", optional = true }
```

Comment:
Can you add another round of unit tests with this feature enabled, just so we don't have any regressions with this feature:

https://github.com/camshaft/bolero/blob/25bba361e00c6948ebbd23273729ae801ef71417/Makefile#L41-L42

```Makefile
unit-tests:
    @cargo test
    @cargo test --features arbitrary
```